From 8c36bdd19fafc517b135f744613dfb10e06475f3 Mon Sep 17 00:00:00 2001
From: Ben Pfaff <blp@nicira.com>
Date: Tue, 10 Jan 2012 14:22:22 -0800
Subject: [PATCH 2/2] debian: Look in /lib/modules instead of /usr/src for
 DKMS kernel sources.

DKMS packages usually look in /lib/modules for kernel sources, since that
is the "standard" location, but our packages was looking directly in
/usr/src.  This fixes the problem.

Reported-by: Alban Browaeys <prahal@yahoo.com>
Tested-by: Alban Browaeys <prahal@yahoo.com>
Signed-off-by: Ben Pfaff <blp@nicira.com>

--- a/debian/dkms.conf.in
+++ b/debian/dkms.conf.in
@@ -1,6 +1,6 @@
 PACKAGE_NAME="openvswitch"
 PACKAGE_VERSION="__VERSION__"
-MAKE="./configure --with-linux=/usr/src/linux-headers-`uname -r` ; make -C datapath/linux"
+MAKE="./configure --with-linux=/lib/modules/`uname -r`/build ; make -C datapath/linux"
 BUILT_MODULE_NAME[0]=openvswitch_mod
 BUILT_MODULE_NAME[1]=brcompat_mod
 BUILT_MODULE_LOCATION[0]=datapath/linux/
-- 
1.7.6.3

